src/ILCompiler.ReadyToRun/src/Compiler/DependencyAnalysis/ReadyToRun/ModuleTokenResolver.cs

```diff
@@ -280,6 +294,8 @@ public DummyTypeInfo GetTypeFromSpecification(MetadataReader reader, ModuleToken
                 throw new NotImplementedException();
             }
         }
+
+        public CompilerTypeSystemContext TypeSystemContext => _typeSystemContext;
```

Comment:
This looks unused. It also seems like a weird place to get hold of a type system context

Reply:
Thanks for spotting, removed (along with one more unused method I noticed above this one).